### ISSUE TITLE
[FeatureRequest] New Command Get-GraphClient that returns the httpclient used by the module.

### ISSUE THREAD

**Issue by JustinGrote** [microsoftgraph/msgraph-sdk-powershell#127]
The current Autorest-based module has several limitations. Please make it easy to work around those limitations by exposing the httpclient used by the module so that we can invoke additional custom queries.

Alternatively, Add a Invoke-Graph -URI <uri> that runs a request using the existing Httpclient and returns the raw output.

This way the graph API token doesn't have to be directly exposed (sure you can still fetch it from the cookies property of the httpclient, but it's not stored securely in memory cross-platform anyways)

Use case: I want to download files with /content, there's no option to do that, and if I try to modify an existing client with an HTTPRequesthandler, the downloaded file is not in json format so the command still fails.

**Comment by darrelmiller** [comment]
Having some kind of Invoke-GraphRequest seems like a good idea.  It also is a good workaround for us being able to get at non-JSON responses.  /cc @peombwa

**Comment by jmiller76** [comment]
@JustinGrote  does 
$Client = [Microsoft.Graph.PowerShell.Authentication.Helpers.HttpHelpers]::GetGraphHttpClient((Get-MgContext))

Do what you need?

**Comment by JustinGrote** [comment]
Yes it does! Thanks for hunting down that extension method for me, I hadn't upgraded to 0.2.0 yet.

I put this function in my profile for convenience:
``` powershell
function Get-GraphClient ($context = (Get-MGContext)) {
  [Microsoft.Graph.PowerShell.Authentication.Helpers.HttpHelpers]::GetGraphHttpClient($context)
}
```

**Comment by BHoggs** [comment]
@JustinGrote I see you closed this issue - despite finding a workaround using the extension method, as per @darrelmiller's comment I still feel the module would benefit from a Invoke-GraphRequest type of Cmdlet? Should this stay open for that purpose?

**Comment by JustinGrote** [comment]
@BHoggs I'd be fine with it if it's within the scope of the team, to use, so we can note that the workaround is sufficient but having it as a cmdlet would be better.

**Comment by darrelmiller** [comment]
@finsharp We should create a wrapper cmdlet called Invoke-GraphRequest that allows arbitrary calls to the Graph with the token.
